src/components/EditableTagField/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from "react";
+import styled from "@emotion/styled";
+import { InstanceTag, ParameterInput } from "gql/generated/types";
+import { convertArrayToObject } from "utils/array";
+import { TagRow } from "./TagRow";
+
+type Tag = InstanceTag | ParameterInput;
+type EditableTagFieldProps = {
+  onChange: (data: Tag[]) => void;
+  inputTags: Tag[];
+  visible?: boolean;
```

Comment:
Can we set it up so that a tag is not only visible if visible=true, but also if visible=null? Aka only hide it if visible=false?

Reply:
This prop serves to reset the component state when the modal rendering it is hidden. I can rename the prop to make it more clear. It does not affect which tags are rendered.